File: src/main/scala/mffs/card/ItemCard.java
package mffs.card;

import mffs.base.ItemMFFS;
import calclavia.api.mffs.card.ICard;

public class ItemCard extends ItemMFFS implements ICard
{
	public ItemCard(int id, String name)
	{
		super(id, name);
		this.setMaxStackSize(1);
	}
}

File: mffs/src/main/scala/com/calclavia/edx/mffs/item/card/ItemCard.scala
package com.calclavia.edx.mffs.item.card

import com.calclavia.edx.mffs.api.card.Card
import com.calclavia.edx.mffs.base.CategoryMFFS
import com.resonant.core.prefab.modcontent.AutoItemTexture
import nova.core.item.Item

class ItemCard extends Item with Card with CategoryMFFS with AutoItemTexture {
	override def getMaxCount: Int = 1

	override def getID: String = "cardBlank"
}